django/db/models/options.py

```diff
@@ -545,6 +545,8 @@ def get_field(self, field_name):
         """
         Return a field instance given the name of a forward or reverse field.
         """
+        if field_name == 'pk':
+            return self.pk
```

Comment:
`FieldDoesNotExist('pk')` is also wrong though because there is never a field named `pk`, that's what this whole changeset is about. I do agree with you though.